pkg/builder/webhook_test.go

```diff
@@ -1214,3 +1346,23 @@ func (*TestCustomDefaultValidator) ValidateDelete(ctx context.Context, obj runti
 }
 
 var _ admission.CustomValidator = &TestCustomValidator{}
+
+type testValidatorDefaulter struct{}
```

Comment:
Do we intentionally have two structs here that both are using TestCustomDefaultValidator internally?